vm/systemSmartContracts/esdt.go

```diff
@@ -1005,6 +1015,8 @@ func (e *esdt) configChange(args *vmcommon.ContractCallInput) vmcommon.ReturnCod
 		return vmcommon.UserError
 	}
 
+	e.statusHandler.SetStringValue(common.MetricESDTIssuanceCost, newConfig.BaseIssuingCost.String())
```

Comment:
You need to initialize this on the constructor also

Reply:
already done when loading configs, but added here as well for safety